Hakuba/Hakuba.swift

```diff
@@ -149,7 +149,7 @@ public extension Hakuba {
     // MARK - Reset
     
     func reset(listType: SectionIndexType.Type) -> Self {
-        let sections = (0...listType.count).map { _ in Section() }
+        let sections = Array(count: listType.count, repeatedValue: Section())
```

Comment:
@ikesyo  Because `Section` is a reference type, so if we use `repeatedValue` method to create an array, all elements in array will refer to the same object instance.